helix-term/src/main.rs

```diff
@@ -76,6 +76,11 @@ FLAGS:
 
     let args = Args::parse_args().context("could not parse arguments")?;
 
+    helix_loader::initialize_config_file(args.config_file.clone());
+    helix_loader::initialize_log_file(args.log_file.clone());
+
+    setup_logging(args.verbosity).context("failed to initialize logging")?;
```

Comment:
Technically it could be moved right before `--health`, I can do that. In terms of future-proofing, I think doing it as soon as possible is best.

Reply:
I see that fetch/build grammars and health do not use logging, but I still moved `setup_logging` to the top. I can leave it where it was if you disagree.